xchange-coinmate/src/main/java/info/bitrich/xchange/coinmate/CoinmateStreamingAdapter.java

```diff
@@ -0,0 +1,67 @@
+package info.bitrich.xchange.coinmate;
+
+import info.bitrich.xchange.coinmate.dto.CoinmateWebSocketUserTrade;
+import info.bitrich.xchange.coinmate.dto.CoinmateWebsocketOpenOrder;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.Order;
+import org.knowm.xchange.dto.marketdata.Trades;
+import org.knowm.xchange.dto.trade.LimitOrder;
+import org.knowm.xchange.dto.trade.OpenOrders;
+import org.knowm.xchange.dto.trade.UserTrade;
+import org.knowm.xchange.dto.trade.UserTrades;
+
+import java.math.BigDecimal;
+import java.sql.Date;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CoinmateStreamingAdapter {
```

Comment:
Does this need to be public?

Reply:
Is it another way,i know only that? Do you mean to add it to CoinmateStreamingTradeService class as a private class? Thanks

Reply:
If it's only used by classes in this package, it can be default visibility (just omit the `public`). If it's used by other packages, don't worry.